wordfugue/blog/feeds.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib.syndication.views import Feed
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```

Comment:
Django, in 2.1, got rid of the `core.urlresolvers` module, and replaced it with `urls`, which means we need to update all the references to the old module.